Curatour/frontend
# Fix/event detail view

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
Add venue data to the event query.
Pass event data to Event Preview from Dashboard AND Calendar.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Lines 19-21 of `src/components/calendar/Calendar.js`
Lines 12-16 of `src/components/calendar/event-utils.js`
Lines 7-23 of `src/components/event/Event.js`

*How should this be manually tested?*
Open application and click on an event. See if that is the right event.
*Any background context you want to provide?*
*What are the relevant tickets?*



## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
